www/source/blog/2018-06-13-base-plans-refresh.html.md

```diff
@@ -0,0 +1,170 @@
+
+Many of them also must be updated simultaneously - if we update gcc to 8.1 and it requires a later version of glibc than we current have, we must also update glibc.
+
+### What else does it affect?
+
+We are simultaneously refreshing nearly every [Core Plan](https://github.com/habitat-sh/Core-Plans/) at the same time as we are refreshing the Base Plans.  This is because the majority (if not all) of the Core Plans depend on at least one of the Base Plans in some fashion (or, at the very least, a Base Plan is in their dependency chain).
+
+## Base Plans and You
+
+So far we've discussed how and why we are refreshing the Base Plans and Core Plans, but what does that mean for the Plans in your own origin?
+
+### How does this affect me?
+
+Chances are, any Plan within your origin depends on at least one of the Core Base Plans somewhere in its chain of dependencies.  When that Plan is updated, your Plan also must be rebuilt.  This sounds simple, but if you have Plans in your origin that depend on other Plans - and those Plans depend on a Core Plan (let's say glibc), all of them must be built with the same version of glibc.  Let's look at this visually.
+
+### Dependencies
+
+Let's say I have a Plan under my origin (nellshamrell) called widget_world - let's say it lives at nellshamrell/widget_world on Builder.
+
+```
+nellshamrell/widget_world
+```
+
+Now, let's say that Plan depends on two more Plans nellshamrell/widget and nellshamrell/world (yes, it's contrived, just go with it for now).
+
+```
+nellshamrell/widget_world
+-> nellshamrell/widget
+-> nellshamrell/world
+```
+
+And let's say each of those Plans depends on core/glibc for some reason
+
+```
+nellshamrell/widget_world
+-> nellshamrell/widget
+   --> core/glibc
+-> nellshamrell/world
+   --> core/glibc
+```
+
+And let's say both are using the current core/glibc Plan, which means they are both using version 2.22
+
+```
+nellshamrell/widget_world
+-> nellshamrell/widget
+   --> core/glibc/2.22
+-> nellshamrell/world
+   --> core/glibc/2.22
+```
+
+My nellshamrell/widget_world Plan will build fine, because both of its dependencies are built with the same version of glibc.
+
+```
+(studio) $ build widget_world
+...
+Success!
+I love when a Plan.sh comes together!
+```
+
+Then suppose the Habitat team upgrades core/glibc to 2.27.  And let's say for some reason you only update one of the dependencies to use the updated glibc - so nellshamrell/widget will depend on core/glibc/2.27, while nellshamrell/world will still depend on core/glibc/2.22
+
+```
+nellshamrell/widget_world
+-> nellshamrell/widget
+   --> core/glibc/2.27
+-> nellshamrell/world
+   --> core/glibc/2.22
+```
+
+This time, if I attempt to build "nellshamrell/widget_world", it will return a very ugly error that looks similar to this:
```

Comment:
Probably remove the quotes around `"nellshamrell/widget_world"`, since it's unquoted elsewhere.